packages/lesswrong/components/seasonal/HomepageCommunityMap.tsx

```diff
@@ -1,38 +1,1204 @@
+    "lat": 47.9981645,
+    "lng": 7.8576265
+  },
+  {
+    "_id": "mo7LkrMAxnh5S3ksr",
+    "lat": 28.6024274,
+    "lng": -81.2000599
+  },
+  {
+    "_id": "RkiEmLe7s5R2tdAn8",
+    "lat": 54.00973759999999,
+    "lng": -2.7859802
+  },
+  {
+    "_id": "qfktAPKrDRiwMv9Am",
+    "lat": 33.960048,
+    "lng": -83.382717
+  },
+  {
+    "_id": "kbQGxGXhckpnstkG8",
+    "lat": 33.4930598,
+    "lng": -112.1113979
+  },
+  {
+    "_id": "sRGECzX6fnfFchrbR",
+    "lat": 44.4289447,
+    "lng": 26.0351844
+  },
+  {
+    "_id": "9JpZ4XHvkrnt6NbBh",
+    "lat": 39.147744,
+    "lng": -86.537783
+  },
+  {
+    "_id": "BrsnfyKFDAzwptJHg",
+    "lat": 40.01907629999999,
+    "lng": -105.1379106
+  },
+  {
+    "_id": "pMjEs8y9CMjHCmKXA",
+    "lat": 31.7493125,
+    "lng": 35.18681249999999
+  },
+  {
+    "_id": "DKMphwNAgcCwgZABQ",
+    "lat": 51.7493921,
+    "lng": -1.2383467
+  },
+  {
+    "_id": "jyRkrCkhwYPzEopH5",
+    "lat": 40.8908622,
+    "lng": -73.3540881
+  },
+  {
+    "_id": "Tjrw8JzgaW9zahZM2",
+    "lat": 43.7194164,
+    "lng": 10.4000338
+  },
+  {
+    "_id": "KERimTihvPXu3bKB7",
+    "lat": 28.6024274,
+    "lng": -81.2000599
+  },
+  {
+    "_id": "cavXhfgn3CrmEpHhn",
+    "lat": 28.6024274,
+    "lng": -81.2000599
   }
+]
+
+const styles = communityMapStyles
+
+const LocalEventWrapperPopUp = ({localEvent, handleClose}:{
+  localEvent:LocalEvent,
+  handleClose: (eventId: string)=>void
+}) => {
+  const { StyledMapPopup, GroupLinks } = Components
+  const { document, loading } = useSingle({
+    documentId: localEvent._id,
+    collectionName: "Posts",
+    fragmentName: 'PostsList',
+  });
+
+  if (!localEvent.lat || !localEvent.lng) return null
+
+  if (loading) return null
+
+  if (!document) return null
+  const { htmlHighlight = "" } = document.contents || {}
+  const htmlBody = {__html: htmlHighlight};
+
+  return <StyledMapPopup
+    lat={localEvent.lat}
+    lng={localEvent.lng}
+    link={postGetPageUrl(document)}
+    title={` [Event] ${document.title} `}
+    metaInfo={document.contactInfo}
+    cornerLinks={<GroupLinks document={document}/>}
+    onClose={() => handleClose(document._id)}
+    offsetTop={-25}
+  >
+    <div dangerouslySetInnerHTML={htmlBody} />
+  </StyledMapPopup>
+}
+const LocalEventWrapperPopUpComponent = registerComponent("LocalEventWrapperPopUp", LocalEventWrapperPopUp);
+
+
+const localEventMapMarkerWrappersStyles = (theme: ThemeType): JssStyles => ({
+  icon: {
+    height: 20,
+    width: 20,
+    fill: theme.palette.event,
+    opacity: 0.8
+  }
+})
+const LocalEventMapMarkerWrappers = ({localEvents, classes}: {
+  localEvents: Array<LocalEvent>,
+  classes: ClassesType,
+}) => {
+  const { LocalEventWrapperPopUp } = Components
+  const [ openWindows, setOpenWindows ] = useState([] as Array<string>)
+  const handleClick = useCallback(
+    (id) => { setOpenWindows([id]) }
+    , []
+  )
+  const handleClose = useCallback(
+    (id) => { setOpenWindows(without(openWindows, id))}
```

Comment:
Do we need to pull in lodash/without here?  If `id` is just a string we're trying to filter out, why not `openWindows.filter(windowId => windowId !== id)`?